src/commands/cluster-myshardid.json

```diff
@@ -0,0 +1,18 @@
+{
+    "MYSHARDID": {
+        "summary": "Return the node shard id",
+        "complexity": "O(1)",
+        "group": "cluster",
+        "since": "7.0.0",
```

Comment:
@madolson which version should I use for `since`?

Reply:
I think the next minor version is 7.2.0

Reply:
Will do

Reply:
Yeah, I think given that it's a rather large change we need to do it in 7.2